daq/utils.py

```diff
@@ -34,3 +38,11 @@ def proto_json(message):
 def dict_proto(message, proto_func, ignore_unknown_fields=False):
     """Convert a standard dict object to a proto object"""
     return json_format.ParseDict(message, proto_func(), ignore_unknown_fields)
+
+
+def write_pid_file(pid_file, logger=None):
+    pid = os.getpid()
+    if logger:
+        logger.info('pid is %d', pid)
```

Comment:
Maybe add path of PID file?  Either way...

Reply:
Added file path in the log message